Sources/Valet/SecureEnclaveValet.swift

```diff
@@ -232,7 +231,9 @@ public final class SecureEnclaveValet: NSObject {
     // MARK: Private Properties
 
     private let lock = NSLock()
-    private let baseKeychainQuery: [String : AnyHashable]
+    private var baseKeychainQuery: [String : AnyHashable] {
+        return service.generateBaseQuery()
+    }
```

Comment:
I don't love this, but `AnyHashable` isn't `Sendable` and `service.generateBaseQuery()` is fast, so in the tug-of-war between "have the compiler check the thing I know is safe" and "we know it's safe so opt out of checking to do something faster" I'm leaning towards the former.